edge-patron/src/main/resources/domain/edge-patron/features/patrons.feature

```diff
@@ -2,78 +2,94 @@ Feature: patron tests
 
   Background:
     * url baseUrl
-    * callonce login testAdmin
-    * configure headers = {  'x-okapi-token': '#(okapitoken)', 'Accept': '*/*'  }
-    * def uuid = call random_uuid
+    * callonce login { tenant: 'diku', name: 'diku_admin', password: 'admin' }
```

Comment:
Why is this using the diku admin user (within the diku tenant) rather than the test admin user (for the tenant created by the tests)?

Reply:
to generate an api key we should have a 'salt' , 'username', 'tenantId' , with all necessary permissions.
But in Karate test's each run tenant name is getting change so i dont think it will be possible with **testTeanant**. 

Other edge-modules for karate tests are using diku tenant only.

Reply:
> Other edge-modules for karate tests are using diku tenant only.

This surprises me, I thought the intention was for these tests to be run within a new tenant each time

Reply:
> But in Karate test's each run tenant name is getting change so i dont think it will be possible with testTeanant.

Why does needing an API key make it not possible to use the test tenant? Can an API key be generated for a (system) user created in the test tenant?

Reply:
> 

yes , we can create but the problem is user will get change again in next run. if we dont want to use **diku** then we need to create an seperate user similar to diku with permissions .